src/firecfg/firecfg.h

```diff
@@ -37,6 +37,15 @@
 #include "../include/common.h"
 #define MAX_BUF 4096
 
+// config files
+#define FIRECFG_CFGFILE SYSCONFDIR "/firecfg.config"
+#define FIRECFG_CONF_GLOB SYSCONFDIR "/firecfg.d/*.conf"
+
+// programs
+#define FIREJAIL_EXEC PREFIX "/bin/firejail"
+#define FIREJAIL_WELCOME_SH LIBDIR "/firejail/firejail-welcome.sh"
+#define FZENITY_EXEC        LIBDIR "/firejail/fzenity"
+#define ZENITY_EXEC "/usr/bin/zenity"
```

Comment:
Looks like this adds a dependency on `zenity` to firejail, correct? Not that I'd mind, but package maintainers might need to get informed on this. Usually adding something to release notes takes care of such a change. Just a FYI.